com.unity.render-pipelines.universal/Runtime/RendererFeatures/ScreenSpaceAmbientOcclusion.cs

```diff
@@ -279,19 +292,27 @@ public override void OnCameraSetup(CommandBuffer cmd, ref RenderingData renderin
                         break;
                 }
 
+                // Set up the descriptors
+                RenderTextureDescriptor descriptor = cameraTargetDescriptor;
+                descriptor.msaaSamples = 1;
+                descriptor.depthBufferBits = 0;
+
+                m_AOPassDescriptor = descriptor;
+                m_AOPassDescriptor.width /= downsampleDivider;
+                m_AOPassDescriptor.height /= downsampleDivider;
+                m_AOPassDescriptor.colorFormat = RenderTextureFormat.ARGB32;
+
+                m_BlurPassesDescriptor = descriptor;
+                m_BlurPassesDescriptor.colorFormat = RenderTextureFormat.ARGB32;
+
+                m_FinalDescriptor = descriptor;
+                m_FinalDescriptor.colorFormat = RenderTextureFormat.R8;
```

Comment:
Do all platforms that support SSAO support R8 format?

Reply:
Good point! 
I'm not sure so I'll add a `SystemInfo.SupportsRenderTextureFormat(R8)` check for that to be on the safe side.